Post/tests.py

```diff
@@ -93,3 +95,101 @@ def form():
             'Description': description,
         }
     )
+
+
+@pytest.mark.django_db
+def test_create_post(client):
+    Post.objects.all().delete()
+
+    new_user = User.objects.create_user(
+        username='Roni', email='Test26@gmail.com', password='password2266'
+    )
```

Comment:
Can you create a fixture for creating users and posts ?

Reply:
@Leeadar you already have a fixdture for creating a user no?
is yes so @Shoval-Ovadia just use it then

Reply:
https://github.com/beyond-io/PicATrip/blob/main/users/tests/test_user.py 😃

Reply:
@Leeadar @guy9050 
Hi!
I added two fixtures as you asked (I didn't use Leead's fixtures because I save code that way).
Can you make a review again?

Reply:
what do you mean by "save code that way"? 
there's already a fixture to create a user so why did you create another one?

Reply:
you can move that fixture to conftest.py (create it if it doesn't exist) to be able to use that

Reply:
Hi @guy9050,
Leead made a fixture of a user and I need two different users.
So what should I do? remove 'second_user' fixture and import Leead's user fixture?

Reply:
yes, remove the second one since it's identical to the first one, and use parameterized fixture